scripts/lib-franklin.js

```diff
@@ -104,6 +104,14 @@ export function getMetadata(name) {
   return meta || '';
 }
 
+export function createMetadata(name, value) {
```

Comment:
IMO this become default and should go into upstream.

Reply:
@basecode with upstream you mean contributing it to the boilerplate?